anda/tools/copyparty/copyparty.spec

```
@@ -0,0 +1,44 @@
+%global debug_package %nil #needed to avoid "empty %files file" errors
+
+Name:           copyparty
+Version:        1.18.5
+Release:        1%?dist
+Summary:        Portable, featureful, and fast file server 
+URL:            https://github.com/9001/copyparty
+Source0:        %url/archive/refs/tags/v%version.tar.gz
+License:        MIT
+BuildRequires:  python3-devel python3-pip
```

We support EL which doesn't automatically pull this alongside `python3-devel` unlike Fedora.
```suggestion
BuildRequires:  python3-devel python3-pip pyproject-rpm-macros
```

---

Thanks for the suggestion, fixed in new commit!